kolibri/core/assets/src/kolibri_core_app.js

```diff
@@ -1,13 +0,0 @@
-'use strict';
-
-var Kolibri = require('./facade/facade');
-
-var logging = require('loglevel');
-
-var kolibri = new Kolibri();
-
-logging.setDefaultLevel(2);
```

Comment:
This functionality does not seem to have been replicated in your de-abstraction above. I would like us to keep logging customizable for debug purposes (I have been meaning to, but have not yet, added a NODE_ENV for compilation here to set the debug level).